src/config/section/compute.js

```diff
@@ -319,6 +319,20 @@ export default {
           dataView: true,
           show: (record, store) => { return ['Destroyed'].includes(record.state) && store.features.allowuserexpungerecovervm }
         },
+        {
+          api: 'unmanageVirtualMachine',
+          icon: 'disconnect',
+          label: 'label.action.unmanage.virtualmachine',
+          args: ['id'],
+          dataView: true,
+          groupAction: true,
+          show: (record) => { return ['Running', 'Stopped'].includes(record.state) && record.hypervisor === 'VMware' },
+          mapping: {
+            id: {
+              value: (record, params) => { return record.id }
```

Comment:
we use mapping when the key is not the same as specified in the `args`, so this will still work without adding the mapping here